webpack/assets/javascripts/react_app/components/Layout/browserSupportHelper.js

```diff
@@ -0,0 +1,41 @@
+import browserUpdate from 'browser-update';
+import { runningInPhantomJS } from '../../common/helpers';
+
+export const checkBrowserSupport = () => {
+  if (runningInPhantomJS()) return;
+  browserUpdate({
+    required: {
+      i: 12, // obsolete IE completely
+      e: -2,
+      f: -5,
+      o: -5,
+      s: -4,
+      c: -5,
+    },
+    text: `
+      <b class="buorg-mainmsg">
+        ${__('We will drop support for your browser ({brow_name}) soon.')}
```

Comment:
I wonder if perhaps we just want to trigger a react component, we can use the 
```
    nomessage: false,
    // Do not show a message if the browser is out of date, just call the onshow callback function
```
this will allow us full control and a bit less cryptic to maintain?

Reply:
Ok, have tried that.